days/day-05/solutions/one-liner.stektpotet.py

```diff
@@ -0,0 +1,2 @@
+import sys; [(print(max(ids)), print(i)) for ids in [{int(''.join('0' if c == 'F' or c == 'L' else '1' for c in k), 2) for k in sys.stdin.read().split('\n')[:-1]}] for i in range(min(ids), max(ids)) if not i in ids]
```

Comment:
Hmm. Is this really a one-liner though? :eyes: I would argue that `import sys;` is on a separate "line" :P 

Ok it is a one-liner, but it is not a one-statementer. It is a two-statementer. Approved!

Reply:
Tell me if you find a way of reading `stdin` wothout using an imported library, and I'll change it to a one-statementer! 🤭